CONTRIBUTING.md

```diff
@@ -2,14 +2,22 @@
 
 Thank you for your interest in contributing!
 
-## Adding a Hacker News implementation
-
 If your Hacker News implementation meets the [specifications](https://github.com/tastejs/hacker-news-pwas#specification), don't hesitate to put up a PR for it!
 
-## Updating the site
-
-We're exploring using wildcards for our handlers in the `appl.yaml` file but if you add a static file or asset to the repository, you'll have to include it in file in the meantime.
+If you would like to file a PR to update the site's design, please file an [issue](https://github.com/tastejs/hacker-news-pwas/issues/new) for it first.
 
-## Testing Service Worker locally
+## Setup
 
-To test the Service Worker locally you can run `npm run dev-sw`.
+1. Fork the repo
+2. Clone your fork
+3. Make a branch for your changes
+4. `npm install`
+5. `npm run dev` to kick off a development server
+6. To test the Service Worker locally you can run `npm run dev-sw` instead.
+7. If you're adding a Hacker News implementation:
```

Comment:
Perhaps we can word this so it's clear we mean a "summary" for a Hacker News implementation rather than their whole implementation? It should be obvious but for folks unaware of the project I think we want to make sure they don't try committing the actual app here :p